src/go/k8s/tests/e2e/scale-up/01-assert.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: scale-up-cluster
+  labels:
+    app.kubernetes.io/name: "redpanda"
+    app.kubernetes.io/instance: "scale-up-cluster"
+    app.kubernetes.io/component: database
```

Comment:
Out of the context of this PR: The database label seems a bit off. Should we go with `redpanda` or sth like `streaming`?

Reply:
yeah I think someone else brought this up as well. I think ultimately we can say whatever there. I'll take a look at some prior art to keep consistent, otherwise yeah, we can go with streaming (but I would do that change in another PR)

Reply:
I think ultimately we can say anything in there. Spark just says "spark" 🤷‍♀️ https://github.com/helm/charts/blob/2f030a8e9db3e451e8030b12d2658371f5708f7d/incubator/jaeger/templates/spark-cronjob.yaml#L9 and cockroach says cockroachdb https://github.com/helm/charts/blob/2f030a8e9db3e451e8030b12d2658371f5708f7d/stable/cockroachdb/README.md#L216

Reply:
Yeah going with `redpanda` seems fine.
Definitely a change to be introduced in a separate PR :+1: